code/modules/tgui/states/cameras.dm

```diff
@@ -0,0 +1,29 @@
+/**
+ * tgui state: camera_state
+ *
+ * Cameras work a bit differently than standard TGUI
+ * windows - they do not use UI_DATA the same way to
+ * update. That is, you can watch cameras at huge ranges
+ * regardless. camera_state disables long range viewing
+ * and observers (they can just go there!) but leaves it
+ * interactive for silicons.
+ *
+ */
+
+GLOBAL_DATUM_INIT(camera_state, /datum/ui_state/camera_state, new)
+
+/datum/ui_state/camera_state/can_use_topic(src_object, mob/user)
```

Comment:
Don't make a new `ui_state`, they're old NanoUI junk.

Override `ui_status` instead.

Reply:
oh! i thought it was the inverse. will do

Reply:
WTF none ever told me that.